helm/hpcc/templates/NOTES.txt

```diff
@@ -53,41 +53,35 @@ This chart has defined the following HPCC components:
 
 {{- range $dali := (.Values.dali | default dict) }}
  {{- if and (not $dali.disabled) (hasKey $dali "services") -}}
-  {{- $sashaServices := $dali.services | default dict -}}
-  {{- if not $sashaServices.disabled -}}
-   {{- range $serviceName, $_service := $sashaServices }}
-    {{- $service := ($_service | default dict) -}}
-    {{- if not $service.disabled -}}
-     {{- if hasKey $names $serviceName -}}
-      {{- required ( printf "Component name %s in section sasha was already seen in section %s" $serviceName (get $names $serviceName)) nil -}}
-     {{- else }}
-  dali.sasha.{{ $serviceName }}
-      {{- $_ := set $names $serviceName (printf "dali.%s" $dali.name) -}}
-      {{- $localImage := include "hpcc.imageName" (dict "root" $ "me" $dali ) | quote }}
-      {{- if not ( eq $defaultImage $localImage ) -}}
-       {{- printf " using overridden image %s" $localImage -}}
-      {{- end }}
-     {{- end -}}
-    {{- end -}}
-   {{- end -}}
-  {{- end -}}
- {{- end -}}
-{{- end -}}
-
-{{- $sashaServices := $.Values.sasha | default dict -}}
-{{- if not $sashaServices.disabled -}}
- {{- range $serviceName, $_service := $sashaServices }}
-  {{- $service := ($_service | default dict) -}}
-  {{- if not $service.disabled -}}
+  {{- $daliSashaServicesCtx := dict "services" $dali.services | default dict -}}
```

Comment:
This would be a bit easier to read with some extra parenthesis showing what the default applied to.

Reply:
agreed, will change.